### ISSUE TITLE
Not getting an expected output

### ISSUE THREAD

**Issue by ShraddhaDevaiya** [awsm-research/PyExplainer#21]
Hello, I am using the pyexplainer to explain the data regarding the bug report. following is my input data : 

<img width="181" alt="image" src="https://user-images.githubusercontent.com/29624037/185660140-4bf6453a-3478-46d9-8891-490069061c32.png">

and the feature data is like this:

<img width="293" alt="image" src="https://user-images.githubusercontent.com/29624037/185660232-d3bff136-c0ec-4726-bfbd-83968ff5b2ae.png">

and for this pyexplainer is giving output like this:

<img width="725" alt="image" src="https://user-images.githubusercontent.com/29624037/185660467-01217a60-cd63-4d42-8264-627db2a4595f.png">

which doesn't seem like an explanation. can you please suggest, am I missing anything?

**Comment by MichaelFu1998-create** [comment]
Hi @ShraddhaDevaiya,
I just tested the current version of the package and found no issues with the front-end components.
There could be a case where the "RuleFit" local model (it is used to generate rules which will be presented in the UI as explanations) used by pyexplainer could not generate rules for your input.
May I ask a few questions
(1) what kind of model did you try to explain?
(2) did you use default parameters when calling the "explain" function?

**Comment by ShraddhaDevaiya** [comment]
Hello, 

the random forest model you are asking, that only I tried to explain, and the explain function I am using like this :

rules = py_explainer.explain(X_explain=X_explain,
                             y_explain=y_explain,
                             search_function='crossoverinterpolation')

Did you ask this?

**Comment by MichaelFu1998-create** [comment]
Hi @ShraddhaDevaiya,
It looks alright to me.

Did you train your own RF model?
What is the approximate size of your training data?

**Comment by ShraddhaDevaiya** [comment]
yes, I trained the RF model. and the training data size is only 10 items. so that could be the issue, means very less data to train the model?

**Comment by MichaelFu1998-create** [comment]
Hi @ShraddhaDevaiya ,
Yes, I think that could be the case.
We trained the RF model using around 9k data points in our original experiments.
However, I'm not sure how much data should be to train the model.
You may grab more data and try if it works.

Also, I find that the current version of pyexplainer can be improved,
the UI should tell the user in the case that the rules can't be generated,
otherwise, it would be confusing like what you've just encountered.
Thanks for opening this issue, I'll improve the UI accordingly.

**Comment by ShraddhaDevaiya** [comment]
Thanks for helping. Let me check with the more number of training data.

**Comment by MichaelFu1998-create** [comment]
Okay, I'll leave this open for a while then
If you still have any concerns, feel free to comment here.

**Comment by ShraddhaDevaiya** [comment]
> 

sure, thanks !

**Comment by ShraddhaDevaiya** [comment]
Hello @MichaelFu1998-create , I have tried with 2k data inputs for training, but still it is giving the same kind of output. It is not showing any explanation.
